ambry-clustermap/src/main/java/com.github.ambry.clustermap/HelixBootstrapUpgradeUtil.java

```diff
@@ -702,6 +711,17 @@ private void maybeAddCluster() {
     }
   }
 
+  private void startClusterManager() throws IOException {
+    Properties props = new Properties();
+    props.setProperty("clustermap.host.name", "localhost");
+    props.setProperty("clustermap.cluster.name", clusterName);
+    props.setProperty("clustermap.datacenter.name", adminForDc.keySet().iterator().next());
+    props.setProperty("clustermap.dcs.zk.connect.strings", Utils.readStringFromFile(zkLayoutPath));
+    ClusterMapConfig clusterMapConfig = new ClusterMapConfig(new VerifiableProperties(props));
+    HelixClusterAgentsFactory helixClusterAgentsFactory = new HelixClusterAgentsFactory(clusterMapConfig, null, null);
```

Comment:
just to confirm, the validating HelixClusterManager is going to talk to one zookeeper in one of the specified datacenters? Here "specified" means the selected DCs which will be updated/affected by Helix tool. And do you think it's worth setting up a second validating HelixClusterManager to simulate those in untouched DCs?